proto/stratos/pot/v1/query.proto

```diff
@@ -37,15 +42,15 @@ service Query {
   }
 
   rpc TotalMinedToken(QueryTotalMinedTokenRequest) returns (QueryTotalMinedTokenResponse) {
-    option (google.api.http).get = "/stratos/pot/v1/total-mined-token";
+    option (google.api.http).get = "/stratos/pot/v1/total_mined_token";
   }
 
   rpc CirculationSupply(QueryCirculationSupplyRequest) returns (QueryCirculationSupplyResponse) {
-    option (google.api.http).get = "/stratos/pot/v1/circulation-supply";
+    option (google.api.http).get = "/stratos/pot/v1/circulation_supply";
```

Comment:
@Xiong-stratos  What context to change to `_`? If it is agreed, you should also fix in big-dipper for these changes.

Reply:
Some of our query url are using "-", and some are using "_"。
The style of query url is Inconsistent.

Reply:
@Xiong-stratos I think the big dipper metrics where calculate the APY used this api